src/Symfony/Component/DependencyInjection/Compiler/ResolveParameterPlaceHoldersPass.php

```diff
@@ -38,7 +38,7 @@ public function process(ContainerBuilder $container)
                 $definition->setFile($parameterBag->resolveValue($definition->getFile()));
                 $definition->setArguments($parameterBag->resolveValue($definition->getArguments()));
                 if ($definition->getFactoryClass(false)) {
-                    $definition->setFactoryClass($parameterBag->resolveValue($definition->getFactoryClass()));
+                    $definition->setFactoryClass($parameterBag->resolveValue($definition->getFactoryClass(false)));
```

Comment:
Yes it is! We still have to trigger the relevant notices, and only them. Silencing can be turned off (and will be by users preparing for 3.0 migration).